src/http/GraphQLHttpServer.ts

```diff
@@ -92,15 +93,17 @@ export class GraphQLHttpServer<TContext> {
   // koa helpers
 
   koaHandler() {
-    return (ctx: KoaContextLike) => this.handleKoa(ctx);
+    return (ctx: KoaContextLike, next: KoaNextFunctionLike) =>
+      this.handleKoa(ctx, next);
   }
 
-  async handleKoa(ctx: KoaContextLike) {
+  async handleKoa(ctx: KoaContextLike, next: KoaNextFunctionLike) {
     const response = await this.execute(ctx.request);
 
     ctx.status = response.status;
     ctx.set(response.headers);
     ctx.body = response.body;
+    await next();
```

Comment:
Do we have guarantees that `next` is always defined in Koa? I believe the current test suite would fail if not, but better safe than sorry :D!

Reply:
Here, https://github.com/koajs/koa/blob/master/docs/guide.md#response-middleware they say

> When the furthest downstream middleware executes next();, it's really yielding to a noop function, allowing the middleware to compose correctly anywhere in the stack.

In any case, I have put the invocation behind a condition now, just to be safe.